RhoInc/safety-explorer-suite
# Safety Explorer Suite v0.8.3

# Summary
- Adds new `chartinit` callback allowing custom code to be run whenever a chart is initialized. 
- Adds a new `\linked-participants` example. 
- Clean up `\docs` page

# Test Notes

- Check the new `test-page\linked-participants` example
   - confirm that clicking a participant updates the header in all charts.
   - confirm that details for a selected participant are shown when the chart is changed (with no additional user input needed) . 
- Confirm that the original `test-page\` has been update to use real (public) Xanomeline data (instead of simulated data)
- Quick QC of the links and content on the `\docs` page
